IFC4x3/Sections/Domain specific data schemas/Schemas/IfcRailDomain/PropertySets/Pset_OnSiteTelecomControlUnit/DocPropertySet.xml

```diff
@@ -0,0 +1,15 @@
+<?xml version="1.0" encoding="utf-8"?>
+<DocPropertySet xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" Name="Pset_OnSiteTelecomControlUnit" UniqueId="1e449be7-399f-a03b-a1a1-ecfa87db1ec2" ApplicableType="IfcController" PropertySetType="PSET_TYPEDRIVENOVERRIDE">
```

Comment:
Checking if it is correct that applicability is for all PDTs?

Reply:
Yes Applicability is for IfcController!